### ISSUE TITLE
mask计算的问题

### ISSUE THREAD

**Issue by litchiyj** [bojone/crf#11]
crf_keras.py 文件的loss如下： 
def loss(self, y_true, y_pred):  # 目标y_pred需要是one hot形式
        if self.ignore_last_label:
            mask = 1 - y_true[:, :, -1:]
        else:
            mask = K.ones_like(y_pred[:, :, :1])

看了一下ConditionalRandomField的代码，https://github.com/bojone/bert4keras/blob/master/bert4keras/layers.py
def dense_loss(self, y_true, y_pred):
        """y_true需要是one hot形式
        """
        # 导出mask并转换数据类型
        mask = K.all(K.greater(y_pred, -1e6), axis=2, keepdims=True)
        mask = K.cast(mask, K.floatx())
dense_loss计算mask用的y_pred， y_pred的值应该都是大于-1000000（-1e6），mask全是1 能起到什么作用呢？
不是应该用y_true来计算mask吗？类似crf_keras.py中的 mask = 1 - y_true[:, :, -1:]
实在没想通，望大神拨冗解惑一下

**Comment by litchiyj** [comment]
看了一下源代码，应该是继承了embedding层的mask，对y_pred已经进行了处理，是可以的
